Fixed `org-odt-data-dir`.

rothmichaels opened the pull request:
`org-odt-data-dir` was being set to an abosulte path for *org-mode*
included with emacs installed in /usr/share.

Updated `org-odt-dat-dir` to point to the *org-mode* pack.

rothmichaels commented:
Forgot about this one; last one for real.

samaaron commented:
Apologies, I'm not quite sure what's going on here... Could you explain a little more?

rothmichaels commented:
Running `org-odt-export-to-odt` requires certain files in the directory referenced by `org-odt-data-dir` (e.g. *OrgOdtStyles.xml*). *Org-pack* was defining this variable to the hard coded path */usr/share/emacs/etc/org*, which is not available to all emacs installs.

The *org-pack* does contain a valid org/odt data directory at *org-pack/lib/org-mode/etc*. 

I made the following changes:
- Removed duplicate definition of `org-odt-data-dir` from *org-pack/lib/org-version.el*.
- Updated definition of `org-odt-data-dir` in *org-pack/lib/org-loaddefs.el* to reference the data directory included with the pack (i.e. *org-pack/lib/org-mode/etc*).

Sorry for the brief pull request description. Let me know if you have any other questions.

rothmichaels commented:
Any thoughts on this one? Got more questions? Here is a super brief summary:

- You had `org-odt-data-dir` defined twice (as the same value).
- `org-odt-data-dir` is defined to be a system specific org-mode path */usr/share/emacs/etc/org* instead of using the org-mode lisp included in the org-mode pack.

rothmichaels commented:
Closing because I later realized this was not the correct place to set this variable.
